src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperOutput.cs

```diff
@@ -33,19 +30,15 @@ internal TagHelperOutput(string tagName)
         /// </summary>
         /// <param name="tagName">The HTML element's tag name.</param>
         /// <param name="attributes">The HTML attributes.</param>
-        /// <param name="htmlEncoder">The <see cref="IHtmlEncoder"/> used
-        /// to encode HTML attribute values.</param>
         public TagHelperOutput(
             string tagName,
-            [NotNull] IDictionary<string, string> attributes,
-            [NotNull] IHtmlEncoder htmlEncoder)
```

Comment:
Ahhh awesome no longer need this. 

You should also remove this: https://github.com/aspnet/Razor/blob/dev/src/Microsoft.AspNet.Razor.Runtime/project.json#L9